### ISSUE TITLE
Is there an option to control display resolution? font size?

### ISSUE THREAD

**Issue by gilzellner** [stestagg/dashcast#1]
We are using this to display our build status and the content is larger than the display size.

**Comment by mrothenbuecher** [comment]
normally the chrome cast decides which resolution it will use based on the monitor it is plugged in.

720p / 1080p

**Comment by gilzellner** [comment]
yeah, but the text could be scaled, the same way you zoom in and out of a webpage on chrome.

**Comment by puco** [comment]
My chromecast only manages 720p. Is there a way how to influence/force the resolution? I have a Chromecast ultra and a 4k TV (and can play 4k YouTube videos) but the viewport is still only 720p.

**Comment by mrothenbuecher** [comment]
Sometimes you need to tell the tv which kind of device is pluged into the hdmi port. (for instance samsung tv's, if you change the source typ to pc it works in 1080 otherwise not)

**Comment by puco** [comment]
The TV correctly shows that the HDMI input is 4k, the videos cast over YouTube are clearly 4k. Just the browser/custom receiver has viewport / clientWidth/clientHeight at 720p

**Comment by mrothenbuecher** [comment]
The problem is not the viewport or other things. [The chromecast decides in which resolution will be send on his hdmi port ](https://www.reddit.com/r/Chromecast/comments/21nz9h/changing_the_resolution_on_the_chromecast/ci1ea0v/)

**Comment by axlroden** [comment]
websites will only be 720p you can't change that.
